url_store.json

```diff
@@ -1354,11 +1354,6 @@
         "test": null,
         "data": null
     },
-    {
-        "url": "https://www.instagram.com/{}/",
-        "test": "method",
-        "data": null
-    },
```

Comment:
I tested it again with a known username and another one completely random and I am not getting a false positive, can you share the usernames you tested for instagram if possible?
![Screenshot_2021-05-27_09-45-23](https://user-images.githubusercontent.com/36354846/119765198-67454500-bed0-11eb-848a-0b08b3eb6d11.png)
![Screenshot_2021-05-27_09-44-39](https://user-images.githubusercontent.com/36354846/119765201-69a79f00-bed0-11eb-825b-bc56da167080.png)

Reply:
this looks like something funky that's happening in the container. maybe the version of python or something? 

On bare metal (mac): 

```sh
$ nexfil.py -u essgeewhy |grep instag
[+] https://www.instagram.com/essgeewhy/
$ 

$ nexfil.py -u notessgeewhy |grep instag
$ 
```

inside the container: 

```sh
$ docker run -it nexfil-upstream -u essgeewhy | grep instag
[+] https://www.instagram.com/accounts/login/
$ 

$ docker run -it nexfil-upstream -u notessgeewhy | grep instag
[+] https://www.instagram.com/accounts/login/
$ 
```

Reply:
I see, I will test more, thanks!